draft-packages/guidance-block/KaizenDraft/GuidanceBlock/GuidanceBlock.tsx

```diff
@@ -97,7 +96,7 @@ class GuidanceBlock extends React.Component<
     actions: GuidanceBlockActions,
     withActionButtonArrow?: boolean
   ) => (
-    <Media query={MOBILE_QUERY}>
+    <Media query="(max-width: 767px)">
```

Comment:
Is it worth leaving a `TODO` comment here to import this media query from a centrally managed point again later?